product-service/src/config/db_init.bat

```diff
@@ -0,0 +1,34 @@
+aws dynamodb put-item --table-name products --item "{\"id\": {\"S\":\"de2dae5c-3b58-4661-b2df-1ee3089091c4\"}, \"description\": {\"S\": \"Action role-playing game, Adventure game\"}, \"price\": {\"N\": \"24.16\"}, \"title\": {\"S\": \"Xenoblade Chronicles 3\"}}"
```

Comment:
Also there is `aws dynamodb batch-write-item` to load multiple items. Those items could be stored in JSON format somewhere. Take a look https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SampleData.LoadData.html